hoomd/md/ForceComposite.cc

```diff
@@ -973,18 +982,64 @@ void ForceComposite::updateCompositeParticles(uint64_t timestep)
         updated_pos += dr_space;
         quat<Scalar> updated_orientation = orientation * local_orientation;
 
+        // update velocity of constituents
+        // central particle angular momentum, velocity, moment of inertia
+        const quat<Scalar> angmom(h_angmom.data[central_idx]);
+        const vec3<Scalar> inertia(h_inertia.data[central_idx]);
+
+        // Calculate the angular momentum in the body frame as a vector,
+        // then divide by the moment of inertia to get the angular velocity.
+        // The tangential velocity in the body frame is the angular velocity
+        // cross the position relative to the center of mass, which is then
+        // rotated into the space frame and added to the velocity of the center
+        // of mass.
+        quat<Scalar> angvel_body_quat = Scalar(0.5) * conj(orientation) * angmom;
+        vec3<Scalar> angvel_body = angvel_body_quat.v;
```

Comment:
For long comments, prefer not to use a bunch of `//`. Also, the quaternion here can also be const, right?
```suggestion
        /* Update velocity of constituents
         *
         * Calculate the angular momentum in the body frame as a vector,
         * then divide by the moment of inertia to get the angular velocity.
         * The tangential velocity in the body frame is the angular velocity
         * cross the position relative to the center of mass, which is then
         * rotated into the space frame and added to the velocity of the center
         * of mass.
         */
        const quat<Scalar> angmom(h_angmom.data[central_idx]);
        const vec3<Scalar> inertia(h_inertia.data[central_idx]);
        const quat<Scalar> angvel_body_quat = Scalar(0.5) * conj(orientation) * angmom;
        vec3<Scalar> angvel_body = angvel_body_quat.v;
```